tenable/io/v3/base/endpoints/explore.py

```diff
@@ -28,7 +28,7 @@ def details(self, obj_id: Union[str, UUID]):
 
             >>> tio.{PATHWAY}.details('00000000-0000-0000-0000-000000000000')
         '''
-        self._get(obj_id).json()
+        self._get(obj_id, _conv_json=self._conv_json)
```

Comment:
_conv_json should be conv_json.  this attribute isn't a private.

https://restfly.readthedocs.io/en/latest/api/session.html#restfly.session.APISession._req

Reply:
Updated